Gillian-C/lib/External.ml

```diff
@@ -1,5 +1,6 @@
 open Gillian.Gil_syntax
 open Gillian.General
+module L = Gillian.Logging.Default
```

Comment:
Not liking this, if you change the logger you change everywhere
Maybe create a Logging module in `Gillian-C` that just contains `include Gillian.Logging.Default`

Reply:
Then you can also just call `Logging` everywhere, which is nicer too